tensorflow-core/tensorflow-core-api/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Script to build native TensorFlow libraries
+set -eu
+
+# Allows us to use ccache with Bazel
+export BAZEL_USE_CPP_ONLY_TOOLCHAIN=1
+export BAZEL_VC="C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/"
+export PYTHON_BIN_PATH=$(which python.exe)
+if [[ ! -f $PYTHON_BIN_PATH ]]; then
+    export PYTHON_BIN_PATH=$(which python3)
+fi
+
+# Build TensorFlow itself
+bazel build --python_path="$PYTHON_BIN_PATH" --config=mkl --output_filter=DONT_MATCH_ANYTHING --verbose_failures @org_tensorflow//tensorflow:tensorflow @org_tensorflow//tensorflow/java:tensorflow
+
+# Normalize some paths with symbolic links
+TENSORFLOW_SO=(bazel-bin/external/org_tensorflow/tensorflow/libtensorflow.so.?.?.?)
+if [[ -f $TENSORFLOW_SO ]]; then
+    ln -sf $(basename $TENSORFLOW_SO) bazel-bin/external/org_tensorflow/tensorflow/libtensorflow.so
+    ln -sf $(basename $TENSORFLOW_SO) bazel-bin/external/org_tensorflow/tensorflow/libtensorflow.so.2
+fi
+TENSORFLOW_DYLIB=(bazel-bin/external/org_tensorflow/tensorflow/libtensorflow.?.?.?.dylib)
+if [[ -f $TENSORFLOW_DYLIB ]]; then
+    ln -sf $(basename $TENSORFLOW_DYLIB) bazel-bin/external/org_tensorflow/tensorflow/libtensorflow.dylib
+    ln -sf $(basename $TENSORFLOW_DYLIB) bazel-bin/external/org_tensorflow/tensorflow/libtensorflow.2.dylib
+fi
+
+# Copy generated Java source files
+mkdir -p src/gen/java/
+cp -r bazel-genfiles/external/org_tensorflow/tensorflow/java/ops/src/main/java/* src/gen/java/
+cp -r bazel-genfiles/external/org_tensorflow/tensorflow/java/_javac/tensorflow/libtensorflow_sourcegenfiles/* src/gen/java/
```

Comment:
This last line should be remove if we run the annotation processor from Maven instead (see [this comment](https://github.com/tensorflow/java/pull/1#issuecomment-536228429))

Reply:
We can, but what does it give us? Until everything is moved to Java, it seems like one more failure point that can get out of sync and that we'll need to watch for.

Reply:
The annotation processor does not depend on any native code and is actually already [working pretty well](https://github.com/tensorflow/java/tree/master/core/annotation-processor) outside Bazel right now, it is pure Java. I don't see any good reason for taking a step back at this stage just to restore it later, let's keep it instead, what we want from the Bazel build is the code generated by the C++ engine only.

Reply:
Just thought of something, we actually **need** the annotation processor to be ran outside Bazel, because otherwise it will not generate bindings based on the [op classes](https://github.com/tensorflow/java/tree/master/core/tensorflow-core/src/main/java/org/tensorflow/op/core) inside our repo but on those found in the old code base of the main repository, which is not what we want.